doc/nrf/protocols/thread/overview/security.rst

```diff
@@ -0,0 +1,212 @@
+.. _ug_ot_thread_security:
+
+OpenThread Security
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section describes the security features of the |NCS| OpenThread implementation.
+By default, the OpenThread stack uses the :ref:`nrf_security` (nrf_security) and `Platform Security Architecture (PSA)`_ for cryptographic operations.
+
+Nordic OpenThread samples leverage :ref:`security` features supported in the |NCS| that can be divided into four major categories:
+
+* Secure processing environment
+* Cryptography
+* Secure storage
+* Platform Security Support
+
+In the following sections you will learn more details about each listed category.
+
+Secure processing environment
+*****************************
+
+Depending on the board, Thread samples can use a secure processing environment.
+
+nRF54L with Trusted Firmware-M (TF-M)
+=====================================
+
+On the nRF54L SoC, Thread samples support :ref:`app_boards_spe_nspe` with Trusted Firmware-M (TF-M).
```

Comment:
```suggestion
On the nRF54L15 SoC, Thread samples support :ref:`app_boards_spe_nspe` with Trusted Firmware-M (TF-M).
```
if it's just MLT (?), if not we should probably say nRF54L Series devices.

Reply:
Currently, it is only for L15.

Reply:
got it, then the suggestion applies :)